site/components/Footer/index.tsx

```diff
@@ -2,68 +2,86 @@ import React, { FC } from "react";
-          <Link href="/">
-            <a>
-              <Trans>Home</Trans>
-            </a>
-          </Link>
-          <Link href="/buy">
-            <a>
-              <Trans>Get KLIMA</Trans>
-            </a>
-          </Link>
-          <a href={urls.stake} target="_blank" rel="noopener noreferrer">
-            <Trans>Stake</Trans>
+export const Footer: FC = () => (
+  <footer className={styles.footer}>
+    <div className={styles.footer_content}>
+      <nav className={styles.footer_nav}>
+        <Link href="/">
+          <a>
+            <Trans>Home</Trans>
           </a>
-          <a href={urls.bonds} target="_blank" rel="noopener noreferrer">
-            <Trans>Bond</Trans>
+        </Link>
+        <Link href="/buy">
+          <a>
+            <Trans>Get KLIMA</Trans>
           </a>
-          <a href={urls.officialDocs} target="_blank" rel="noreferrer noopener">
-            <Trans>Docs</Trans>
+        </Link>
+        <a href={urls.stake}>
+          <Trans>Stake</Trans>
+        </a>
+        <a href={urls.bonds}>
+          <Trans>Bond</Trans>
+        </a>
+        <a href={urls.officialDocs} target="_blank" rel="noreferrer noopener">
+          <Trans>Docs</Trans>
+        </a>
+        <Link href="/blog">
+          <a>
+            <Trans>Blog</Trans>
           </a>
-          <Link href="/blog">
-            <a>
-              <Trans>Blog</Trans>
-            </a>
-          </Link>
-          <Link href="/contact">
-            <a>
-              <Trans>Contact</Trans>
-            </a>
-          </Link>
-          <Link href="/disclaimer">
-            <a>
-              <Trans>Disclaimer</Trans>
-            </a>
-          </Link>
-        </nav>
-
-        <nav className={styles.footer_icons}>
-          <a href={urls.twitter} target="_blank" rel="noreferrer noopener">
-            <TwitterIcon />
-          </a>
-          <a href={urls.github} target="_blank" rel="noreferrer noopener">
-            <GithubIcon />
+        </Link>
+        <Link href="/contact">
+          <a>
+            <Trans>Contact</Trans>
           </a>
-          <a
-            href={urls.discordInvite}
-            target="_blank"
-            rel="noreferrer noopener"
-          >
-            <DiscordIcon className="discordIcon" />
+        </Link>
+        <Link href="/disclaimer">
+          <a>
+            <Trans>Disclaimer</Trans>
           </a>
-        </nav>
-      </div>
-    </footer>
-  );
-};
+        </Link>
+      </nav>
+
+      <nav className={styles.footer_icons}>
+        <a href={urls.twitter} target="_blank" rel="noreferrer noopener">
+          <TwitterIcon />
+        </a>
+        <a href={urls.youtube} target="_blank" rel="noreferrer noopener">
+          <YoutubeIcon />
+        </a>
+        <a href={urls.discordInvite} target="_blank" rel="noreferrer noopener">
+          <DiscordIcon />
+        </a>
+        <a href={urls.reddit} target="_blank" rel="noreferrer noopener">
```

Comment:
I keep saying to myself I should create a `<A>` component that does `target` and `rel` by default :) lets do that in a follow up at some point

Reply:
Thought I might as well include it into this PR since I'm in the space, turned out to be quite the rabbit hole retrofitting the new component 😅